fdbserver/worker.actor.cpp

```diff
@@ -2605,6 +2705,9 @@ ACTOR Future<Void> workerServer(Reference<IClusterConnectionRecord> connRecord,
 					ASSERT(req.initialClusterVersion >= 0);
 					LocalLineage _;
 					getCurrentLineage()->modify(&RoleLineage::role) = ProcessClass::ClusterRole::Storage;
+
```

Comment:
There aren't many comments in this new code, and since it involves changes in several places it took a bit for me to piece together how it works.  I think there should at least be comments
- Right here explaining the approach and why this is the trigger point.  
- Where storageCleaners is declared, to explain what it contains and the when things would be added to / removed from it.

Reply:
sure thing. added more comments. thank you!